amplipi/app.py

```diff
@@ -79,6 +81,8 @@
 
 app.add_exception_handler(NotAuthenticatedException, not_authenticated_exception_handler)
 
+logger = logging.getLogger('ampli-logger')
+logger.setLevel(logging.DEBUG)
```

Comment:
We shouldn't just hard-code DEBUG log level, this should ideally be a config option.